### DISCUSSION TITLE
Meta data columns. What is VERSION for?

### DISCUSSION THREAD

**Discussion by daniel-ls-howard** [nextgenhealthcare/connect#5998] [Q&A] [answered]
I notice that channels have Default Metadata Columns for TYPE, SOURCE and VERSION. What does version show? I have a test channel and I don't see any data in there. 

TIA

**Comment by pacmano1** [comment, marked as answer]
HL7 Version.

![image](https://github.com/nextgenhealthcare/connect/assets/44065187/2f58977f-73b2-4f9f-977c-57e5b0c94427)
